cmd/validator/flags/flags.go

```diff
@@ -345,10 +345,10 @@ var (
 		Value: params.BeaconConfig().EthBurnAddressHex,
 	}
 
-	// EnableValidatorRegistrationFlag enables the periodic validator registration API calls that will update the custom builder with validator settings.
-	EnableValidatorRegistrationFlag = &cli.BoolFlag{
-		Name:  "enable-validator-registration",
-		Usage: "Enables validator registration APIs (MEV Builder APIs) for the validator client to update settings such as fee recipient and gas limit",
+	// EnableBuilderFlag enables the periodic validator registration API calls that will update the custom builder with validator settings.
```

Comment:
Note this is a breaking change so we'll need to give @Taranpreet26311 and wonderful develop team a heads up